pkg/defaults/accesscontrol/role.go

```diff
@@ -34,11 +34,14 @@ const (
 	// TODO: ROX-14398 Remove default role VulnReporter
 	// VulnReporter is a role that has the minimal privileges required to create and manage vulnerability reporting configurations.
 	VulnReporter = "Vulnerability Report Creator"
+
+	// ConfigController is a role that grants the config-controller exactly the access it needs
+	ConfigController = "Configuration Controller"
```

Comment:
PolicyReconciler? ConfigReconciler?

Reply:
Policy vs Config: I am wanting to keep the name generic in case we add more CRDs in the future (which has not been agreed upon yet).  

Controller vs Reconciler:  From the [kubebuilder book](https://book.kubebuilder.io/cronjob-tutorial/controller-overview.html#whats-in-a-controller), a controller is the executable component that makes the current state looks like the desired state, and reconciliation is the process/function inside the controller that does that.  Based on those definitions, I think it's more appropriate to use the term controller in this context.